llvm/tools/llvm-cov/SourceCoverageView.h

```diff
@@ -266,6 +268,10 @@ class SourceCoverageView {
   /// digits.
   static std::string formatCount(uint64_t N);
 
+  uint64_t Count1(uint64_t N) const { return (N && BinaryCounters ? 1 : N); }
```

Comment:
This does more than count ones, can you give it a more descriptive name?

Reply:
I prefer shorter names since they (`Count1` and `formatCount1`) are replacements of `Count` and `formatCount`.
That said, they appear in fewer lines. `BinaryCount`/`formatBinaryCount`?

Reply:
BinaryCount + formatBinaryCount SGTM

Reply:
Done.